opass

```diff
@@ -325,7 +330,10 @@ def main():
     def do_git_svn_thing(thing):
         # THING is probably "rebase", "push", or "dcommit"
         try:
-            completed = subprocess.run([pass_pgm, 'git-svn', thing,],
+            args = [pass_pgm, 'git-svn', thing,]
+            if verbose:
+                print(args)
```

Comment:
Can there be some window dressing around this?  That is, instead of just printing the raw args, print a status message that explains where we are ("About to execute 'pass git-svn {thing}...") and then shows the args, so that the user has some context?

Reply:
Addressed in 04df5a5f17c9388a7c7fb3a79d7b6717f33cfe83